snippets/go.json

```diff
@@ -155,6 +155,18 @@
 		"test function": {
 			"prefix": "tf",
 			"body": "func Test$1(t *testing.T) {\n\t$0\n}"
+		},
+		"init function": {
+			"prefix": "finit",
```

Comment:
sigh.. This takes me back to a programmer's hardest task. http://imgur.com/y3dhJFQ : Naming things.

Going with `fmain` and `finit`